daily_data/show_data.html.erb

```diff
@@ -1,7 +1,10 @@
- <ul><% @all_daily_data.each do |daily_data| %> 
-   <li> On this day <b><%= daily_data.date %></b>, <%= daily_data.step_count %>, <%= daily_data.notes %></li>
- <% end %>
- </ul>
+<%= content_tag(:ul) do %>
+  <% @all_daily_data.each do |daily_data| %> 
+    <%= content_tag(:li) do %>
+      On this day <%= content_tag(:b) do %> <%= daily_data.date %><% end %>, <%= daily_data.step_count %>, <%= daily_data.notes %>
```

Comment:
You can provide the content for [`content_tag`](https://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-content_tag) as a string instead of a block if it’s more convenient. In this case I think it’s nicer to say `<%= content_tag(:b, daily_data.date) %>`.